src/openforms/dmn/tests/test_api.py

```diff
@@ -1,5 +1,7 @@
 from unittest.mock import patch
 
+from django_camunda.dmn.datastructures import DMNIntrospectionResult
+from django_camunda.dmn.types import DMNInputParameter, DMNOutputParameter
```

Comment:
importing the camunda library in a module/plugin that has nothing to do with Camunda is a sign there's something missing in the abstraction layer :smile: